### DISCUSSION TITLE
Book info not saving

### DISCUSSION THREAD

**Discussion by runningfree73** [yermak/AudioBookConverter#392] [Q&A] [answered]
After reopening a converted file, I find the information I previously entered into the "book info" panel hasn't been saved.  The entries are either blank or the information has changed.

I filled out all the entries on the book info panel except for the year. After reopening the file, the title was changed to the same name as the series and the book number was missing.

I just checked the file properties and it looks as though the information has been saved to file so it might just be the app is not reading that information correctly.

**Comment by yermak** [comment, marked as answer]
Created issue #414
